src/ProjectTemplates/test/RazorClassLibraryTemplateTest.cs

```diff
@@ -1,62 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
What do you guys think about this? There's a bunch of packages \ projects that do not ship from the wasm branch. The template tests in particular are some of the longer running tests. Removing them makes a significant dent in the build times from what I could tell locally.

Reply:
If you're going to remove these tests you'll have to ensure the following going forward:
1. Ensure that no non-Blazor API surface-area changes on this branch. In the past when we've lost Template test coverage they tend to end up in a broken state mostly because an API changed and nobody reacted for templates.
2. Make SUPER sure that when we merge Blazor-WASM back into master that the removals here get reverted. My biggest concern with this course of action is that this will get missed when we merge and we won't notice for however long. I'd want some sort of assurance here other than "we'll remember", like the issue to merge the two having a check-list which includes reverting this as a step.

As long as we can get guarantees on those two points I think it's reasonable to remove these for the blazor-wasm branch only.